scripts/apps/authoring/track-changes/views/inline-comments-widget.html

```diff
@@ -11,7 +11,8 @@ <h4 class="label">{{commentObj.fieldName}}</h4>
                     <div class="flex-row sibling-spacer-10">
                         <sd-user-avatar data-user="users[item.data.authorId]"></sd-user-avatar>
                         <div>
-                            <div class="text" sd-comment-text data-name="{{users[item.data.authorId].display_name || users[item.data.authorId].username}}" data-text="{{item.data.msg.split('(user:')[0]}}"></div>
+                            <b>{{users[item.data.authorId].display_name || users[item.data.authorId].username}}:</b>
```

Comment:
Can we modify the back-end so `display_name` defaults to username if not provided. Isn't it already like that @petrjasek ?

Reply:
it should be like that already

Reply:
great, then you can remove the fallback @devketanpro